src/main/scala/org/phenoscape/kb/Term.scala

```diff
@@ -464,14 +464,14 @@ object Term {
     import scalaz._
     val terms = iris.map(iri => sparql" $iri ").fold(sparql"")(_ |+| _)
     val queryText: QueryText = sparql"""
-    SELECT DISTINCT ?term ?term_label
+    SELECT ?term (SAMPLE(?term_lbl) AS ?term_label)
```

Comment:
Why do we need to `sample` ? Why is `distinct` not sufficient? Shouldn't `distinct` remove the redundancy?

Reply:
We are getting multiple different results for `rdfs:label`. We just want to take one for any given subject.

Reply:
Questioning the use of sample still seems to be a valid question, though. Specifically:
- in the example in #197, not all labels are different, which makes one wonder how DISTINCT let those through;
- if there can actually be different labels, whether differing in spelling only or otherwise, doesn't the use of sample mean I could get different results for different invocations?

Reply:
The main issue is that there are multiple representations of the same label, because of inconsistencies across ontologies (`xsd:string`, `@en`, etc). They were appearing the same in the previous results. Yes, it is possible you may get a different result with different invocations, for example if we have both an underscore and a space version.

Reply:
So if we consider them all equivalent, wouldn't it be better to, for example, just take the first. Or the last. Or something else that's determinate rather than random.

Reply:
I see, I think this can be changed to `MIN`.

Reply:
`MIN` is good because it favors space-separated labels over the ones with underscores.